youtube_dl/extractor/rtvnh.py

```diff
@@ -0,0 +1,40 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+
+
+class RTVNHIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?rtvnh\.nl/video/(?P<id>[0-9]+)'
+    _TEST = {
+        'params': {
+            'hls_prefer_native': True
```

Comment:
If it can be downloaded with native hls without any problem it should be forced in `_extract_m3u8_formats` and removed from test.